appveyor.yml

```diff
@@ -0,0 +1,22 @@
+platform: x64
+environment:
+ matrix:
+  - DC: dmd
+    DVersion: nightly
+    arch: x64
+  - DC: dmd
+    DVersion: nightly
+    arch: x86
+skip_tags: false
+build_script:
+ - ps: wget https://raw.githubusercontent.com/wilzbach/installer/appveyor-script/script/install.ps1 -OutFile install.ps1
```

Comment:
Ah nice, this is what I improvised in the mean time.
https://github.com/MartinNowak/io/blob/master/.appveyor.yml
Having a proper PowerShell install script would be great (https://stackoverflow.com/a/40491432/2371032).

Reply:
@MartinNowak can we get dlang/installer#275 merged?